src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator is now working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");

```

Comment:
Similarly, the message 'Excavator has stopped working.' can be improved for clarity by specifying the type of machine, such as 'Excavator machine has stopped working.'